bin/pytorch_inference/CCommandParser.cc

```diff
@@ -203,21 +194,25 @@ void CCommandParser::jsonToRequest(const rapidjson::Document& doc) {
 
     // wipe any previous
     m_Request.s_SecondaryArguments.clear();
-    TUint64Vec arg;
     while (doc.HasMember(varArgName)) {
-        const rapidjson::Value& v = doc[varArgName];
-        for (auto itr = v.Begin(); itr != v.End(); ++itr) {
-            arg.push_back(itr->GetUint64());
-        }
 
-        m_Request.s_SecondaryArguments.push_back(arg);
-        arg.clear();
+        const rapidjson::Value& arr = doc[varArgName];
+        TUint64Vec arg;
+        arg.reserve(m_Request.h * m_Request.w);
+        for (auto itr = arr.Begin(); itr != arr.End(); ++itr) {
```

Comment:
nit: I believe GetArray supports range based for loops so this could just be
```
for (const auto& innerArray : arr.GetArray()) {
    for (const auto &token : innerArray.GetArray()) {
        ...
```
This comment applies to all iterations over rapidjson arrays in this PR.